### PR TITLE
Add visually-hidden example

### PR DESCRIPTION
**Pull request by cloud-walker**
Add example for VisuallyHidden #939

### CODE REVIEW

**Comment by diegohaz** [comment]
Thanks @cloud-walker! I wish we could come up with an example closer to real-world usage; something people would use `VisuallyHidden` for. Do you have anything in mind?

**Comment by cloud-walker** [comment]
Hi @diegohaz, sure!

**Comment by cloud-walker** [comment]
Done it, I've borrowed an icon from the toolbar example.

I've had the tentation to describe more the thing, but then I thought would be better to stick with the current style of the examples.

File: packages/ariakit/src/visually-hidden/__examples__/visually-hidden/test.tsx [hunk 1]
```diff
@@ -0,0 +1,21 @@
+import { getByRole, getByText, render } from "ariakit-test";
+import { axe } from "jest-axe";
+import Example from ".";
+
+test("ally", async () => {
```
Comment (diegohaz) [inline packages/ariakit/src/visually-hidden/__examples__/visually-hidden/test.tsx]:
```suggestion
test("a11y", async () => {
```

File: packages/ariakit/src/visually-hidden/__examples__/visually-hidden/test.tsx [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+  const { container } = render(<Example />);
+  expect(await axe(container)).toHaveNoViolations();
+});
+
+test("render properly", () => {
+  render(<Example />);
+
+  expect(getByRole("button", { name: "Undo" })).toBeInTheDocument();
```
Comment (diegohaz) [inline packages/ariakit/src/visually-hidden/__examples__/visually-hidden/test.tsx]:
```suggestion
  render(<Example />);
  expect(getByRole("button", { name: "Undo" })).toBeInTheDocument();
```

**Review by diegohaz — DISMISSED** [review]
Thanks! This looks good. I made just a few suggestions.

We'll eventually need a more detailed description of each component/example. These will live in the [component documentation](https://github.com/ariakit/ariakit/blob/main/contributing.md#writing-the-component-documentation) (for default examples) and [example documentation](https://github.com/ariakit/ariakit/blob/main/contributing.md#writing-documentation-for-other-examples). I'm working on the website right now so those docs will look nicer.

I'll ping you in case you want to contribute to those docs for the examples you created (or other components/examples).

**Review by diegohaz — APPROVED** [review]

**Pull request merged**